src/requirements/reqs-functions.ts

```diff
@@ -270,10 +270,7 @@ export function computeRequirements(
     })
     .filter((it): it is UserChoiceOnFulfillmentStrategy => it != null);
 
-  const {
-    requirementFulfillmentGraph,
-    illegallyDoubleCountedCourses,
```

Comment:
Did you remove `illegallyDoubleCountedCourses` because it's not fleshed out yet?

Reply:
Because it's triggering a linter warning since it's unused. It will be naturally added back once I implemented the warnings UI.